tests/commands/method.py

```diff
@@ -186,7 +187,8 @@ def runTest(self):
         self.assert_parse("cdrom", "cdrom\n")
 
         # harddrive
-        self.assert_parse("harddrive --dir=/install --biospart=part", "harddrive --dir=/install --biospart=part\n")
```

Comment:
This appears to be unrelated to the method deprecation.

Reply:
Well, yes and no. When `--biospart` was removed for F33, the `harddrive` test was updated, but not the `method` one, which stayed stuck at the older command version and apparently all was well. Now that is no longer the case.

Reply:
Oh! I see now, thanks!